config/spotbugs/filter.xml

```diff
@@ -59,4 +59,8 @@
         <Class name="~software\.amazon\.smithy\.plugin\.language\.SmithySyntaxHighlighter"/>
         <Bug pattern="NP_NONNULL_RETURN_VIOLATION"/>
     </Match>
+    <Match>
+        <Class name="~software\.amazon\.smithy\.plugin\.language\.SmithySyntaxHighlighter"/>
```

Comment:
Looks like this is referring to a file that doesn't exist in the repo yet so this change should remain local to your environment.

Reply:
This file exists here: https://github.com/awslabs/smithy-intellij/blob/main/src/main/java/software/amazon/smithy/plugin/language/SmithySyntaxHighlighter.java

Reply:
Thanks, I didn't realize this was not the main smithy repo. All good